packages/async/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@shopify/async",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "Primitives for loading parts of an application asynchronously.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "tsc --p tsconfig.build.json",
+    "prepublishOnly": "yarn run build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "author": "Shopify Inc.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/quilt.git"
+  },
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
+  },
+  "homepage": "https://github.com/Shopify/quilt/blob/master/packages/async/README.md",
+  "dependencies": {},
+  "devDependencies": {
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/types": "^7.3.0",
+    "@types/babel__core": "^7.0.4",
+    "@types/babel__traverse": "^7.0.5",
+    "@types/webpack": "^4.4.24",
+    "typescript": "~3.0.1"
+  },
+  "files": [
```

Comment:
Where did the webpack plugin go?
Is it being added to the [webpack-asset-metadata-plugin](https://github.com/Shopify/sewing-kit/pull/1140/files) instead?

Reply:
Correct, moved it to this PR: https://github.com/Shopify/sewing-kit/pull/1140